### DISCUSSION TITLE
Using Accordion inside of Menu

### DISCUSSION THREAD

**Discussion by dtrinh100** [primefaces/community#142] [PrimeVue]
I want to create a menu similar to Reddit's popover menu where they have an accordion inside of the menu, alongside with the regular menu items. Is this possible using the Menu and Accordion components or is there a better approach to this?

**Comment by tugcekucukoglu** [comment]
You can customize the Menu and Accordion components to create your own menu component with new pass-through props, using or not unstyled mode, as you wish. `v-styleclass` directive can help with enter and leave animations.

Also similar made with PrimeVue. For example https://blocks.primevue.org/#/application/sidebarnav

**Comment by vikasm-mittal** [comment]
I have a similar case where I want to add the accordion control inside of some other elements, I am able to style it and make the UI work, but what I am noticing is that accordion control only look for direct descendants when creating the children list of type accordion tab, so if I have a tab inside a div it will not be added to the list of tabs, so one problem it causes it is if you set multiple=false in which only one tab can be opened does not work, so you can put 3 tabs inside a tab inside accordion and say multiple=false and still be able to keep all three open.

I am also building a directive to save the states of all tabs and trying to use the active index and active index change but since accordion do not see any indirect descendants tabs nether of the functionality works

@ContentChildren(AccordionTab) tabList: QueryList<AccordionTab> | undefined;
@ContentChildren(AccordionTab, {descendants: true}) tabList: QueryList<AccordionTab> | undefined;
